internal/graph/iterators.go

```diff
@@ -0,0 +1,78 @@
+package graph
```

Comment:
Why not in storage?

Reply:
no super strong opinion but mostly because right atm this is only scoped/used here, and pkg vs internal.